docs/README.md

```diff
@@ -186,7 +186,25 @@ A Pod running on AWS Fargate automatically mounts an Amazon EFS file system, wit
 
 #### Set up driver permission
 The driver requires IAM permission to talk to Amazon EFS to manage the volume on user's behalf. There are several methods to grant driver IAM permission:
-* Using the EKS Pod Identity Add-on - [Install the EKS Pod Identity add-on to your EKS cluster](https://docs.aws.amazon.com/eks/latest/userguide/pod-id-agent-setup.html). This doesn't need the efs-csi-driver to be installed through EKS add-on, it can be used no matter the method of installation of the efs-csi-driver. If this installation method is used, the ```AmazonEFSCSIDriverPolicy``` policy has to be added to the cluster's node group's IAM role. 
+* Using the EKS Pod Identity Add-on - [Install the EKS Pod Identity add-on to your EKS cluster](https://docs.aws.amazon.com/eks/latest/userguide/pod-id-agent-setup.html). This doesn't need the efs-csi-driver to be installed through EKS add-on, it can be used no matter the method of installation of the efs-csi-driver. If this installation method is used, the **AWS managed policy**  ```AmazonEFSCSIDriverPolicy``` has to be added to the IAM role which will be associated with the **k8s service account** of the driver over ```--set controller.serviceAccount.name```. This [Pod Identity's IAM role trust relationship](https://docs.aws.amazon.com/eks/latest/userguide/pod-id-association.html) is described as follows:
```

Comment:
> it can be used no matter the method of installation of the efs-csi-driver

AWS documentation says that Pod Identity is not available for CSI addons installed via managed EKS addon

Reply:
@kkopachev I think your mention is correct, mine approach is self-installation via **HELM**.